### PR TITLE
fix: Line fetch some value null

### PR DESCRIPTION
**Pull request by BestBurning**
fix: Line fetch some value null

### CODE REVIEW

File: hugegraph-loader/src/main/java/com/baidu/hugegraph/loader/reader/file/OrcFileLineFetcher.java [hunk 1]
```diff
@@ -123,11 +123,18 @@ public Line fetch() throws IOException {
...
-
+        Object[] values = null;
         this.row = this.recordReader.next(this.row);
-        Object[] values = this.inspector.getStructFieldsDataAsList(this.row)
-                                        .stream().map(Object::toString)
-                                        .toArray();
+        try {
+            values = this.inspector.getStructFieldsDataAsList(this.row)
+                    .stream()
+                    .map(o -> o != null ? o.toString() : "")
```
Comment (coderzc) [inline hugegraph-loader/src/main/java/com/baidu/hugegraph/loader/reader/file/OrcFileLineFetcher.java:131]: Filter null value？

File: hugegraph-loader/src/main/java/com/baidu/hugegraph/loader/reader/file/OrcFileLineFetcher.java [hunk 2]
```diff
@@ -123,11 +123,18 @@ public Line fetch() throws IOException {
...
+                    .toArray();
+        } catch (NullPointerException e) {
```
Comment (javeme) [inline hugegraph-loader/src/main/java/com/baidu/hugegraph/loader/reader/file/OrcFileLineFetcher.java:133]: I think it won't throw NullPointerException if we translated null to "", right?

**Review by javeme — COMMENTED** [review]
Thank you for your contribution

**Comment by javeme** [comment]
Looks the same as PR https://github.com/apache/incubator-hugegraph-toolchain/pull/321

**Comment by imbajin** [comment]
close it since no update for long time

**Pull request closed without merging**